fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerIndexFromIndexTest.java

```diff
@@ -1150,4 +1157,45 @@ void testIndexFromIndexBlock() {
         assertReadable(tgtIndex);
         scrubAndValidate(List.of(tgtIndex));
     }
+
+    @Test
+    void testIndexFromIndexIgnoreSyncLock() {
+
+        final long numRecords = 180;
+
+        Index srcIndex = new Index("src_index", field("num_value_2"), EmptyKeyExpression.EMPTY, IndexTypes.VALUE, IndexOptions.UNIQUE_OPTIONS);
+        Index tgtIndex = new Index("tgt_index", field("num_value_3_indexed"), IndexTypes.VALUE);
+        FDBRecordStoreTestBase.RecordMetaDataHook hook = myHook(srcIndex, tgtIndex);
+
+        populateData(numRecords);
+
+        openSimpleMetaData(hook);
+        buildIndexClean(srcIndex);
+        disableAll(List.of(tgtIndex));
+
+        openSimpleMetaData(hook);
+
+        IntStream.rangeClosed(0, 4).parallel().forEach(id -> {
+            snooze(100 - id);
+            try {
+                try (OnlineIndexer indexBuilder = newIndexerBuilder(tgtIndex)
+                        .setIndexingPolicy(OnlineIndexer.IndexingPolicy.newBuilder()
```

Comment:
Is this slow enough that it doesn't end up just being one builds and the others see that it is already built?

Same goes for the other tests.

Reply:
Most of the time - yes. 
(Checked the logs messages)